### PR TITLE
Create endpoint to edit user profile info by Logist

### PR DESCRIPTION
**Pull request by AndriiChepeliuk**
--- Added ---

- added controller to edit user profile info;
- added method to get user Id by user Email;

### CODE REVIEW

File: API/Controllers/UserController.cs [hunk 1]
```diff
@@ -37,6 +37,16 @@ public async Task<ActionResult> UserEditProfileInfo(UserEditProfileInfoDTO userE
             return Ok();
         }
 
+        [HttpPost("user-edit-info")]
+        [AuthorizeByRole(IdentityRoleNames.Logist)]
+        public async Task<ActionResult> UserEditProfileInfo(UserEditProfileInfoDTO userEditProfileInfo, string email)
+        {
+
```
Comment (Grasmus) [inline API/Controllers/UserController.cs]: Remove empty line

File: API/Controllers/UserController.cs [hunk 2]
```diff
@@ -51,5 +61,14 @@ public async Task<ActionResult> GetAllUsersAsync()
         {
             return Ok(await _userService.GetAllUsersAsync());
         }
+
+        [HttpDelete("delete")]
+        [AuthorizeByRole(IdentityRoleNames.Logist)]
+        public async Task<ActionResult> DeleteAsync(string email)
+        {
+            await _userService.DeleteUserAsync(email);
+
```
Comment (Grasmus) [inline API/Controllers/UserController.cs]: Remove empty line

File: Core/Resources/ErrorMessages.resx [hunk 3]
```diff
@@ -228,4 +228,7 @@
   <data name="InvalidOrder" xml:space="preserve">
     <value>Invalid point's order!</value>
   </data>
-</root>
+  <data name="NotEnoughPermissions" xml:space="preserve">
+    <value>You are not allowed to do this!</value>
+  </data>
+</root>
```
Comment (Grasmus) [inline Core/Resources/ErrorMessages.resx]: Add empty line

**Review by vladtymo — APPROVED** [review]

File: API/Controllers/UserController.cs [hunk 4]
```diff
@@ -37,6 +37,15 @@ public async Task<ActionResult> UserEditProfileInfo(UserEditProfileInfoDTO userE
             return Ok();
         }
 
+        [HttpPost("user-edit-info")]
+        [AuthorizeByRole(IdentityRoleNames.Logist)]
+        public async Task<ActionResult> UserEditProfileInfo(UserEditProfileInfoDTO userEditProfileInfo, string email)
```
Comment (marynakernychna) [inline API/Controllers/UserController.cs]: I think you should add "Async" at the end of the name of the asynchronous method

**Review by Grasmus — APPROVED** [review]

**Review by Grasmus — APPROVED** [review]

**Pull request merged**